test.py

```diff
@@ -169,6 +169,10 @@ def test_subscriber_names(self):
     def test_subscribe(self):
         self.qiwis.subscribe("app1", "ch3")
         self.assertIn("app1", self.qiwis._subscribers["ch3"])
+        # The app already subscribes to the channel.
+        orgSubscribers = self.qiwis._subscribers["ch3"]
+        self.qiwis.subscribe("app1", "ch3")
+        self.assertEqual(self.qiwis._subscribers["ch3"], orgSubscribers)
```

Comment:
Good! Why don't you separate this as another test method, e.g., `test_subscribe_duplicate()`?

Reply:
It looks more eligible! I will do that.